modules/distribution-service-streaming-wowza/src/main/java/org/opencastproject/distribution/streaming/wowza/WowzaStreamingDistributionService.java

```diff
@@ -276,49 +235,76 @@ public void activate(BundleContext bundleContext) {
       }
 
       logger.info("Streaming distribution directory is {}", distributionDirectory);
-    }
-  }
 
-  public String getDistributionType() {
-    return DISTRIBUTION_TYPE;
-  }
+      // Streaming URLs
+      List<Organization> organizations = organizationDirectoryService.getOrganizations();
+      for (Organization org: organizations) {
+        String orgId = org.getId();
+        String streamingUrl = StringUtils.trimToNull((String) properties.get(String.format(WOWZA_URL_KEY, orgId)));
+        String streamingPort = StringUtils.trimToNull((String) properties.get(String.format(WOWZA_PORT_KEY, orgId)));
+
+        if (streamingUrl != null) {
+          try {
+            URI tenantStreamingUrl = getStreamingUrl(streamingUrl, streamingPort, validSchemes, DEFAULT_SCHEME, null);
+
+            if (tenantStreamingUrl == null) {
+              throw new ComponentException(String.format("Streaming URL is undefined for tenant %s.", orgId));
+            }
+
+            streamingUrls.put(orgId, tenantStreamingUrl);
+            logger.info("Wowza Streaming URL for tenant {} set to \"{}\"", orgId, tenantStreamingUrl);
+          } catch (URISyntaxException e) {
+            throw new ComponentException(
+                    String.format("Wowza Streaming URL %s of tenant %s could not be parsed", streamingUrl, orgId), e);
+          }
+        } else {
+          logger.warn("Wowza Streaming URL is undefined for tenant {}", orgId);
```

Comment:
Wouldn't that be a valid configuration?
This is basically warning people that they aren't using streaming which doesn't make sense, does it?
I suggest dropping this to debug level.

Reply:
Done.